File: client/src/app/_models/hr/userAttachment.ts

export interface IUserAttachment {
     id: number;
     candidateId: number;
     appUserId: number;
     attachmentType: string;
     attachmentSizeInBytes: number;
     name: string;
     url: string;
     dateUploaded: Date;
     uploadedByEmployeeId: number;
}

export class UserAttachment implements IUserAttachment {
     id=0;
     candidateId=0;
     appUserId=0;
     attachmentType='';
     attachmentSizeInBytes = 0;
     name='';
     url = '';
     dateUploaded = new Date();
     uploadedByEmployeeId = 0;
}

File: api/Entities/HR/UserAttachment.cs
namespace api.Entities.HR
{
    public class UserAttachment: BaseEntity
    {
        public int CandidateId { get; set; }
        public int AppUserId { get; set; }
        public string AttachmentType { get; set; }
        public long Length { get; set; }
        public string Name { get; set; }
        public string UploadedLocation { get; set; }
        public string UploadedbyUserName { get; set; }
        public DateTime UploadedOn { get; set; }
    }
}